gtsam_unstable/examples/TimeOfArrivalExample.cpp

```diff
@@ -57,7 +57,7 @@ vector<Event> createTrajectory(int n) {
   vector<Event> trajectory;
   double timeOfEvent = 10;
   // simulate emitting a sound every second while moving on straight line
-  for (size_t key = 0; key < n; key++) {
+  for (size_t key = 0; key < static_cast<size_t>(n); key++) {
```

Comment:
Why not simply change the argument type to size_t ?

Reply:
Ahh yeah looking at the inputs for when these methods are called, `n` is of type `size_t`.

Reply:
Pushed the fix out.